### DISCUSSION TITLE
Add Amazon Linux 2023 platform to meta.json

### DISCUSSION THREAD

**Discussion by DriesCruyskens** [ansible/ansible-lint#3294] [Ideas]
Since there is a `Amazon` and  `Amazon Linux 2` platform I was wondering wether there should be a `Amazon Linux 2023` one too since it was released recently.

**Comment by konstruktoid** [comment]
Hi @DriesCruyskens, I believe this is a https://github.com/ansible/galaxy issue.

**Reply by DriesCruyskens** [reply]
You're right sorry, it's already an issue:
https://github.com/ansible/galaxy/issues/3141
